Algorithmia/datafile.py

```diff
@@ -33,7 +33,7 @@ def get(self):
         return self.client.getHelper(self.url)
 
     # Get file from the data api
-    def getFile(self):
+    def getFile(self, as_path=False):
```

Comment:
also added a default null flag to `getFile` so that we can actually skip the `.name` weirdness and stop having dangling file handles 🎉